### PR TITLE
Add CIS 1.1 scored scripts. Verify scripts pending

### PR DESCRIPTION
**Pull request by bordoloih**
All scored scripts in CIS 1.1. Corresponding verify scripts are pending

### CODE REVIEW

File: roles/cis/tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+- import_tasks: 1/task.yml
```
Comment (satya-rajesh) [inline roles/cis/tasks/main.yml:2]: There is a typo here (correct to 1/tasks.yml) due to which below error is coming:
1.30s$ docker run --privileged --rm  -v$(pwd):/src ansible-security-hardening:latest /src/examples/remediate_and_verify.sh
/src /
ERROR! Unable to retrieve file contents
Could not find or access '/src/1/task.yml' on the Ansible Controller.

Overall looks good to me.

However unless verification of these cases are added, the travis ci will mostly fail.

**Review by satya-rajesh — COMMENTED** [review]
Overall looks good to me.

Please ensure the travis ci build is not broken:
https://travis-ci.org/vmware/ansible-security-hardening/builds/644550076?utm_source=github_status&utm_medium=notification

**Review by satya-rajesh — COMMENTED** [review]
Please fix below aswell:

vmwclabot Pending — CLA must be signed before merge can proceed

**Comment by ppadmavilasom** [comment]
Lets please make ci pass on each commit.

**Pull request closed without merging**